build.gradle

```diff
@@ -26,3 +26,11 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+ext {
+    roomVersion = '2.2.5'
+    archLifecycleVersion = '2.2.0'
+    coreTestingVersion = '2.1.0'
+    materialVersion = '1.1.0'
+    coroutines = '1.3.4'
+}
```

Comment:
This file needs to be removed. To be clear, I mean removed from GitHub, not from your local project.